### PR TITLE
Add bundler config

### PR DESCRIPTION
**Pull request by zsluedem**

### CODE REVIEW

File: src/bundler/bundler.rs [hunk 1]
```diff
@@ -1,5 +1,37 @@
+use std::str::FromStr;
+
+use clap::Parser;
+use ethereum_types::U256;
+use ethers::types::Address;
+
 use crate::models::wallet::Wallet;
 
+#[derive(Debug, Parser, PartialEq)]
+pub struct BundlerOpts {
```
Comment (Vid201) [inline src/bundler/bundler.rs:10]: I think this should be only Opts, we using full import path in the bundler.rs: aa_bundler::bundler::bundler::Opts

File: src/bundler/bundler.rs [hunk 2]
```diff
@@ -1,5 +1,37 @@
+use std::str::FromStr;
+
+use clap::Parser;
+use ethereum_types::U256;
```
Comment (Vid201) [inline src/bundler/bundler.rs:4]: I think we should switch to ethers::type fully, I had some problems with ethereum_types in the past (ethers::types::U256)

File: src/bundler/bundler.rs [hunk 3]
```diff
@@ -9,3 +41,39 @@ impl Bundler {
...
+
+#[cfg(test)]
+mod test {
+    use super::BundlerOpts;
+    use clap::Parser;
+    use ethereum_types::{Address, U256};
+    use std::str::FromStr;
+
+    #[test]
+    fn test_bundle_opt() {
```
Comment (Vid201) [inline src/bundler/bundler.rs:53]: This is my personal preference, but I do not write prefix `test_`before test names. What do you think? We should have the same naming across codebase (decide for the one).

File: bin/bundler.rs [hunk 4]
```diff
@@ -31,6 +31,13 @@ pub struct Opt {
 
     #[clap(long, default_value = "127.0.0.1:4337")]
     pub rpc_listen_address: String,
+
+    // execution client rpc endpoint
+    #[clap(long)]
+    pub network: SocketAddr,
```
Comment (Vid201) [inline bin/bundler.rs:37]: `pub eth_client_address: SocketAddr` would be better name IMO

File: src/bundler/bundler.rs (hunk 3 above)
Comment (zsluedem) [reply src/bundler/bundler.rs:53]: > We should have the same naming across codebase (decide for the one).

Agree.

So you prefer just `bundle_opt`?

File: bin/bundler.rs (hunk 4 above)
Comment (zsluedem) [reply bin/bundler.rs:37]: I copied the name from [typescript bundler](https://github.com/eth-infinitism/bundler/blob/main/packages/bundler/src/runBundler.ts#L88) and I agree that your name is better.

File: src/bundler/bundler.rs (hunk 3 above)
Comment (Vid201) [reply src/bundler/bundler.rs]: Yes.

**Review by Vid201 — APPROVED** [review]

**Pull request merged**